rabbit_monitor.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+#   coding: UTF-8
+import logging
+import sys
+from time import sleep
+
+import requests
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry
+from structlog import wrap_logger
+
+import settings
+
+__version__ = "0.0.1"
+
+logging.basicConfig(level=settings.LOGGING_LEVEL,
+                    format=settings.LOGGING_FORMAT)
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+class RabbitMonitor():
```

Comment:
I don't agree with this. If we had multiple polling classes, I could see the value in having them inherit from a parent `PollingMonitor` that encompassed any shared logic, but as this is quite heavily tailored for hitting the management console api, I think `RabbitMonitor` should still stand.